hathor/transaction/resources/transaction.py

```diff
@@ -54,6 +54,9 @@ def get_tx_extra_data(tx: BaseTransaction) -> Dict[str, Any]:
                     spent_outputs[index] = spent_tx.hash_hex
                     break
 
+    # Maps the token uid to the token_data value
+    token_uid_map: Dict[bytes, int] = {settings.HATHOR_TOKEN_UID: 0}
```

Comment:
I feel you're not updating this map in your loop.

Reply:
Done, thanks